model/Image.php

```diff
@@ -233,10 +247,10 @@ public function SetRatioSize($width, $height) {
 		$heightRatio = $height / $this->height;
 		if( $widthRatio < $heightRatio ) {
 			// Target is higher aspect ratio than image, so check width
-			if($this->isWidth($width) && !Config::inst()->get('Image', 'force_resample')) return $this;
+			if($this->isWidth($width)) return $this;
```

Comment:
SemVer means that the expectations of the API are not broken; In this case, the expectation is that this method returns an object with the correct image sized. The expectation has nothing to do with HOW that is achieved internally. The promise to be kept is the API, not the specific implementation. 

For instance, [this](https://github.com/gitlabhq/gitlabhq/issues/7072) is not a semver violation.